test/Microsoft.Extensions.Logging.Test/LoggerFilterTest.cs

```diff
@@ -58,6 +58,96 @@ public void ChangingConfigReloadsDefaultFilter()
         }
 
```

Comment:
Do all the filter methods (whether instance methods or extension methods) have at least one or two tests each? (For whatever interesting combinations of params they have.)